server/server.go

```diff
@@ -304,25 +302,28 @@ func NewServer(cfg *config.Config, driver IDriver) (*Server, error) {
 
 func cleanupStaleSocket(socket string) error {
 	sockStat, err := os.Stat(socket)
-	if err == nil {
-		if sockStat.Mode().Type() != os.ModeSocket {
-			return fmt.Errorf(
-				"the specified socket file %s is a %s instead of a socket file",
-				socket, sockStat.Mode().String())
-		}
+	// Todo:According to the previous logic, When it's occur error will return nil?
+	if err != nil {
+		//return fmt.Errorf("stat socket file %s failed: %v", socket, err)
```

Comment:
Remove this line, plz.

Yes it is nil, since this is an deleting operation, and it is silent about the errors.